src/EditorFeatures/CSharpTest/CommentSelection/CSharpToggleLineCommentCommandHandlerTests.cs

```diff
@@ -336,7 +336,7 @@ class C
 {
     void M()
     {
-[|        //var i = 1;
+        [|//var i = 1;
```

Comment:
The start marker of the selection should remain as it was - immediately left of "var".